caddyfile.go

```diff
@@ -23,18 +23,18 @@ func newCommandFromDispenser(d *caddyfile.Dispenser) (cmd Cmd, err error) {
 // parseHandlerCaddyfileBlock configures the handler directive from Caddyfile.
 // Syntax:
 //
-//   exec [<matcher>] [<command> [<args...>]] {
-//       command     <text>
-//       args        <text>...
-//       directory   <text>
-//       timeout     <duration>
-//       log         <log output module>
-//       err_log     <log output module>
-//       foreground
-//       startup
-//       shutdown
-//   }
-//
+//	exec [<matcher>] [<command> [<args...>]] {
```

Comment:
Can you kindly indent here with 4 spaces instead of tab? It makes it a bit more consistent.

Thanks.

Reply:
Yes, I just let the go language tools format the file and it did this, but I'll adjust it :+1:

Reply:
oh interesting, if that is the case, we can leave as is.
previously, Go format does not mess wit the comments.

Reply:
ya, i basically never manually adjust formatting ever - i just let the default language tools do it for me and leave it up to the project to define config if they want to have preferences for styles :)